### PR TITLE
Create a workflow that enables the user to enter the BMC credentials.

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by zyoung51** [comment]
+1

**Comment by tannoa2** [comment]
@uppalk1 , @amymullins @jlongever @zyoung51 @keedya

File: lib/graphs/bootstrap-bmc-credentials-setup-graph.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    tasks: [
+        {
+            label: 'boot-graph',
+            taskDefinition: {
+                friendlyName: 'Boot Graph',
+                injectableName: 'Task.Graph.Run.Boot',
+                implementsTask: 'Task.Base.Graph.Run',
+                options: {
+                    graphName: 'Graph.BootstrapUbuntuMocks',
```
Comment (yyscamper) [inline lib/graphs/bootstrap-bmc-credentials-setup-graph.js]: :-1: I don't think it is a good idea to reference the mock graph here, the mock graph is only used in debugging, you should use the non-mocked one `GraphBootstrapUbuntu`

File: lib/graphs/bootstrap-bmc-credentials-setup-graph.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+// Copyright 2015, EMC, Inc.
```
Comment (yyscamper) [inline lib/graphs/bootstrap-bmc-credentials-setup-graph.js]: copyright 2016

**Comment by yyscamper** [comment]
:+1: after replace the mock graph with a non-mocked one.

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
Test this please

**Comment by tannoa2** [comment]
Test this please

**Comment by tannoa2** [comment]
Test this please

**Comment by jlongever** [comment]
+1

**Pull request merged**